hydroshare/settings.py

```diff
@@ -425,3 +425,59 @@
         'rest_framework.authentication.SessionAuthentication',
     )
 }
+
+####################
+# LOGGING SETTINGS #
+####################
+
+LOGGING = {
+    'version': 1,
+    'disable_existing_loggers': False,
+    'formatters': {
+        'verbose': {
+            'format' : "[%(asctime)s] %(levelname)s [%(name)s:%(lineno)s] %(message)s",
+            'datefmt' : "%d/%b/%Y %H:%M:%S"
+        },
+        'simple': {
+            'format': '[%(asctime)s] %(levelname)s %(message)s',
+            'datefmt' : "%d/%b/%Y %H:%M:%S"
+        },
+    },
+    'handlers': {
+        'syslog': {
+            'level': 'DEBUG',
+            'class':'logging.handlers.RotatingFileHandler',
+            'filename': '/var/log/hydroshare/system.log',
+            'formatter': 'simple',
+            'maxBytes': 1024*1024*15, # 15MB
+            'backupCount': 10,
+        },
+        'djangolog': {
+            'level': 'DEBUG',
+            'class':'logging.handlers.RotatingFileHandler',
+            'filename': '/var/log/hydroshare/django.log',
+            'formatter': 'verbose',
+            'maxBytes': 1024*1024*15, # 15MB
+            'backupCount': 10,
+        },
+        'appslog': {
+            'level': 'DEBUG',
+            'class':'logging.handlers.RotatingFileHandler',
+            'filename': '/var/log/hydroshare/apps.log',
+            'formatter': 'verbose',
+            'maxBytes': 1024*1024*15, # 15MB
+            'backupCount': 10,
+        },
+    },
+    'loggers': {
+        'django': {
+            'handlers':['syslog', 'djangolog'],
+            'propagate': True,
+            'level':'DEBUG',
+        },
+        'apps': {
+            'handlers': ['syslog', 'appslog'],
+            'level': 'DEBUG',
+        },
```

Comment:
I say just get rid of `"apps"` let the module-level loggers be defined as needed. They don't have to be defined here to be used, and doing so is almost never done unless there are really specific reasons to handle one logger differently.

We don't need to include configuration for a logger to create and use it, so no need to go and do so ahead of time when we haven't been presented with a reason to do so. We should put the name of the logger in the default log format, so that even if they're all going to one place we can still identify and separate them.